### PR TITLE
Remove context ref case check from INFO messages

### LINKED ISSUE #724: Validate is expecting uppercase context references (context_ref_case_mismatch)
### Checked for duplicates

Yes - I've already checked

### 🐛 Describe the bug

When I ran validate on a bundle with verbosity flag of 1 (severity level = INFO), it reported `info.label.context_ref_case_mismatch`, for example:

 `Context reference name case mismatch. Value: 'Cassini-Huygens' Expected one of: '[CASSINI-HUYGENS]'
 INFO  [info.label.context_ref_found]  ...Context product found: 'urn:nasa:pds:context:instrument_host:spacecraft.co'`
`Context reference name mismatch. Value: 'Cassini Orbiter Ultraviolet Imaging Spectrograph' Expected one of: '[ULTRAVIOLET IMAGING SPECTROGRAPH]'`
`INFO  [info.label.context_ref_case_mismatch]  ...Context reference name case mismatch. Value: 'Saturn Rings' Expected one of: '[SATURN RINGS]'`
`INFO  [info.label.context_ref_case_mismatch]  ... Context reference type case mismatch. Value: 'Ring' Expected one of: '[RING]'`

### 🕵️ Expected behavior

I don't think context references should use uppercase as Validate is noting, but rather should be using title case (as the labels are currently).

### 📜 To Reproduce

validate bundle_name -R pds4.bundle -v 1 -r bundle_validate.log

### 🖥 Environment Info

_No response_

### 📚 Version of Software Used

Validate v3.1.1

### 🩺 Test Data / Additional context

_No response_

### 🦄 Related requirements

🦄 #xyz

### ⚙️ Engineering Details

_No response_

### PR DESCRIPTION
**Pull request by al-niessner**
# 🗒️ Summary
Remove the case testing

## ⚙️ Test Data and/or Report
No new test data was given. Regression test should pass (automated tests below)

## ♻️ Related Issues
Closes #724

### CODE REVIEW

**Comment by al-niessner** [comment]
Two checks now fail because they look for this mismatch. Cleaned them up and trying again. Historical tickets are #15 and #47.

**Review by nutjob4life — APPROVED** [review]
👍

File: src/main/java/gov/nasa/pds/tools/validate/rule/pds4/ContextProductReferenceValidationRule.java [hunk 1]
```diff
@@ -267,39 +266,6 @@ public void checkContextReferences() throws XPathExpressionException {
...
-                if (names != null && types != null) {
-                  rgp = rgProds.get(rgProds.indexOf(lidvidObj));
-                  rgpNames = rgp.getNames();
-                  rgpTypes = rgp.getTypes();
-                  // check the name
-                  for (String name : names) {
-                    if (!rgpNames.stream().anyMatch(name::equalsIgnoreCase)) {
-                      getListener().addProblem(new ValidationProblem(
-                          new ProblemDefinition(ExceptionType.INFO,
-                              ProblemType.CONTEXT_REFERENCE_FOUND_MISMATCH,
```
Comment (jordanpadams) [inline src/main/java/gov/nasa/pds/tools/validate/rule/pds4/ContextProductReferenceValidationRule.java:283]: @al-niessner This looks like we removed the wrong messages `CONTEXT_REFERENCE_FOUND_MISMATCH`? We still want to check this when we ignore case. I think the problem type we no longer want to catch is `CONTEXT_REFERENCE_FOUND_CASE_MISMATCH`.

**Review by jordanpadams — CHANGES_REQUESTED** [review]

**Comment by jordanpadams** [comment]
Closing this as wontfix. Miscommunication on my part about what should be removed. Linked ticket is duplicate of https://github.com/NASA-PDS/validate/issues/631

**Pull request closed without merging**